### DISCUSSION TITLE
Qfield Imagery

### DISCUSSION THREAD

**Discussion by alexhogan18** [opengisch/QField#2892] [Q&A]
I am trying to upload imagery to use on Qfield on a tablet, the file is a geotiff and is 1.5 gb in size, there is an error when I try to open the project on qfield, just wondering if this is because of the file size or type and if there is a way to use this imagery on the tablet?

**Comment by nirvn** [comment]
The level of details provided here isn't quite sufficient to provide answers. What does the error says? Can you provide screenshots / screencast of the issue?

**Reply by alexhogan18** [reply]
error: some layers have not been packaged correctly on QfieldCloud, These layers may be misconfigured or their data source is not accessible from the QfieldCloud Server, please check the logs of the latest packaging job on qfield.could website

**Comment by gi921120** [comment]
Try exporting the Geotiff to a Geopackage in QGIS. That will resize the layer and create a mosaic inside of the gpkg. Use this gpkg instead of the Geotiff but don't specify this as a basemap.
